File: prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_exercise/godzilla_vs_kong.py
price = float(input())
statists = int(input())
price2 = float(input())
discount = 0
decor = price * 0.10

if statists > 150:
    discount = price2 * 0.10
    price2 = (price2 - discount) * statists

else:
    price2 = (price2 * statists)

end_sum = decor + price2

if end_sum > price:
    money = end_sum - price
    print("Not enough money!")
    print("Wingard needs " + "{:.2f}".format(money) + " leva more.")

elif end_sum <= price:
    money = price - end_sum
    print("Action!")
    print("Wingard starts filming with " + "{:.2f}".format(money) + " leva left.")

File: JS/JavaScript/JS-Basics/ConditionalStatementsExercise/GodzillavsKong.js
function movie(inputs) {
    let budget = parseFloat(inputs[0]);
    let statist = Number(inputs[1]);
    let pricePerStatist = parseFloat(inputs[2]);

    let decor = budget * 0.10;
    let discount = 0;

    if (statist > 150) {
        discount = 0.10;
    }

    let sumWithoutDiscount = (statist * pricePerStatist);
    let sum = sumWithoutDiscount - sumWithoutDiscount * discount + decor;

    if (budget >= sum) {
        console.log(`Action!\nWingard starts filming with ${(budget - sum).toFixed(2)} leva left.`);
    }
    else {
        console.log(`Not enough money!\nWingard needs ${(sum - budget).toFixed(2)} leva more.`);
    }



}

movie(["9587.88",
"222",
"55.68"])
